sql/analyzer/resolve_subqueries_test.go

```diff
@@ -219,44 +219,6 @@ func TestResolveSubqueryExpressions(t *testing.T) {
 				plan.NewResolvedTable(table, db, nil),
 			),
 		},
-		{
-			name: "columns qualified",
```

Comment:
Why did we need to remove this test?

Reply:
forcing resolveSubqueries to stop at default rules introduces one regression in column pruning, because `pruneTables` escape hatches subquery expressions. so this test returns its input, it doesn't work. but we also have query plan tests for this. also i don't like these tests. i figured subquery expression pruning is probably an OK temporary performance regression

Reply:
Makes sense; thanks for the extra context